README.md

```diff
@@ -30,33 +31,112 @@ global:
+* [`postcss`](https://github.com/postcss/postcss-loader): Post-process CSS with Autoprefixer and [other PostCSS plugins](https://github.com/postcss/postcss#built-with-postcss)
+* [`autoprefixer`](https://github.com/passy/autoprefixer-loader): Add vendor prefixes to CSS rules using values from Can I Use
+* [`namespace-css`](https://github.com/jeffling/namespace-css-loader): Namespace your css with a given selector (for encapsulating all rules in one subset of your site)
+
+**translation**
+* [`po`](https://github.com/dschissler/po-loader): Loads a PO gettext file and returns JSON
+
+**support**
+* [`mocha`](https://github.com/webpack/mocha-loader): do tests with mocha in browser or node.js
+* [`eslint`](https://github.com/MoOx/eslint-loader): PreLoader for linting code using ESLint.
+* [`jshint`](https://github.com/webpack/jshint-loader): PreLoader for linting code.
+* [`jscs`](https://github.com/unindented/jscs-loader): PreLoader for style checking.
+* [`injectable`](https://github.com/jauco/webpack-injectable): Allow to inject dependencies into modules
+* [`transform`](https://github.com/webpack/transform-loader): Use browserify transforms as loader.
+* [`image-size`](https://github.com/patcoll/image-size-loader): Loads an image and returns its dimensions and type
+
+For the full list of loaders, please see [list of loaders](http://webpack.github.io/docs/list-of-loaders.html)
 
 ## Support
 
-webpack supports **AMD and CommonJS** module styles. It performs clever static analysis on the AST of your code. It even has an evaluation engine to evaluate simple expressions. This allows you to **support most existing libraries**.
+Webpack supports **AMD and CommonJS** module styles. It performs clever static
+analysis on the AST of your code. It even has an evaluation engine to evaluate
+simple expressions. This allows you to **support most existing libraries**.
 
 ## Code Splitting
 
-webpack allows you to split your codebase into multiple chunks. Chunks are loaded **on demand**. This reduces the initial loading time.
+Webpack allows you to split your codebase into multiple chunks. Chunks are
+loaded asynchronously at runtime. This reduces the initial loading time.
+
+[Code Splitting documentation](http://webpack.github.io/docs/code-splitting.html)
 
 ## Optimizations
 
-webpack can do many optimizations to **reduce the output size**. It also can make your chunks **cache friendly** by using hashes.
+Webpack can do many optimizations to **reduce the output size of your
+javascript** by deduplicating frequently used modules, minifying, and giving
+you full control of what is loaded initially and what is loaded at runtime
+through code splitting. It can also can make your code chunks **cache
+friendly** by using hashes. 
 
+[Optimization documentation](http://webpack.github.io/docs/optimization.html)
 
 
 # A small example of what's possible
```

Comment:
Same here. I never found that example convincing because it demonstrates already too much.